plugins/parsers/carbon2/README.md

```diff
@@ -0,0 +1,3 @@
+# carbon2 data format parser
+
+TODO
```

Comment:
:question:

Reply:
Made the PR marked as `WIP`, this should explain it :)

Reply:
I converted it to draft

Reply:
Added a short README